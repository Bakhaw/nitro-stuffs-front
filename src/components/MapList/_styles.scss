.MapList {
  display: grid;
  grid-template-columns: repeat(auto-fill, 360px);
  gap: 50px;
  justify-content: center;
  padding: 20px 0;

  @include desktop {
    grid-template-columns: repeat(3, 290px);
  }
}
