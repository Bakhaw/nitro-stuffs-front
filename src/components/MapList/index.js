import React from 'react';

import maps from '../../maps';
import MapCard from '../MapCard';

function MapList() {
  return (
    <ul>
      {maps.map(map => (
        <li key={map.key}>
          <MapCard map={map} />
        </li>
      ))}
    </ul>
  );
}

export default MapList;
