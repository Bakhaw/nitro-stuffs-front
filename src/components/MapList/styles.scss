.MaplistContainer {
  display: flex;
  flex: 1;
  flex-wrap: wrap;
  justify-content: center;
  padding-top: 20px;
}
