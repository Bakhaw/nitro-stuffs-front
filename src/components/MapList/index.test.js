import React from 'react';
import { shallow } from 'enzyme';

import MapList from '.';

test('should render properly', () => {
  const container = shallow(<MapList />);
  expect(container.debug()).toMatchSnapshot();
});
