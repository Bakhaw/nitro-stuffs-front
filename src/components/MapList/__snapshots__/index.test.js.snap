// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render properly 1`] = `
"<ul>
  <li>
    <MapCard map={{...}} />
  </li>
  <li>
    <MapCard map={{...}} />
  </li>
  <li>
    <MapCard map={{...}} />
  </li>
  <li>
    <MapCard map={{...}} />
  </li>
  <li>
    <MapCard map={{...}} />
  </li>
  <li>
    <MapCard map={{...}} />
  </li>
  <li>
    <MapCard map={{...}} />
  </li>
  <li>
    <MapCard map={{...}} />
  </li>
  <li>
    <MapCard map={{...}} />
  </li>
</ul>"
`;
